### PR TITLE
Support JSX Fragment syntax for jsx-wrap-multiline.

### LINKED ISSUE #124: jsx-wrap-multiline false positive with React.Fragment shorthand tag
```tsx
const jsx = (
  <>
    <div>
      Test
    </div>
  </>
)
```

The above snippet causes a false positive warning from `jsx-wrap-multiline` -- it wants the `<div>` child to be wrapped with parens, however this is not necessary as the parent is already jsx.

I don't know how the rule works, but probably `jsx-wrap-multiline` just needs to also accept the new AST node type as also being JSX.

`<>` / `</>` support was released on TS 2.6.2, and supported at runtime by React 16.2.0.

### PR DESCRIPTION
**Pull request by dmiller9911**
Fixes #124

Added a check for ts.syntaxKind.JsxFragment in jsx-wrap-multiline.  A newer version of `tsutils` has this check as well, but I was unsure of the policy for updating dependencies.  If you would rather upgrade I will do that instead.

Add tests to verify positive and negative cases. Upgrade ts devDependency to 2.6.2.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tslint-react](https://github.com/palantir/tslint-react), @dmiller9911! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: src/rules/jsxAlignmentRule.ts [hunk 1]
```diff
@@ -68,20 +68,21 @@ function walk(ctx: Lint.WalkContext<void>) {
...
         attributes: Array<ts.JsxAttribute | ts.JsxSpreadAttribute> // TS <=2.2
-            | { properties: Array<ts.JsxAttribute | ts.JsxSpreadAttribute> }, // TS 2.3
+            | { properties: Array<ts.JsxAttribute | ts.JsxSpreadAttribute> } // TS 2.3
+            | ts.JsxAttributes, // TS 2.6
         elementClose: ts.LineAndCharacter,
         closingTag?: ts.JsxClosingElement,
     ) {
-        attributes = Array.isArray(attributes) ? attributes : attributes.properties;
-        if (attributes.length === 0) {
+        const attrs = Array.isArray(attributes) ? attributes : attributes.properties;
```
Comment (dmiller9911) [inline src/rules/jsxAlignmentRule.ts]: TS 2.6.2 did not like the reassignment of attributes.  Setting to a new variable and adding the typing resolved the type checks.

**Comment by adidahiya** [comment]
> A newer version of tsutils has this check as well, but I was unsure of the policy for updating dependencies. If you would rather upgrade I will do that instead.

Yes please, feel free to upgrade `tsutils`

File: package.json [hunk 2]
```diff
@@ -34,6 +34,6 @@
     "path": "^0.12.7",
     "tslint": "^5.5.0",
     "tslint-language-service": "^0.9.6",
-    "typescript": "~2.4.2"
+    "typescript": "~2.6.2"
```
Comment (adidahiya) [inline package.json]: :+1:

**Comment by dmiller9911** [comment]
Will wait for https://github.com/ajafff/tsutils/pull/17 before updating tsutils.

**Comment by adidahiya** [comment]
@dmiller9911 can you please pull in master to get the new CI setup and required checks? Also enable CI on your branch if you haven't yet.
